<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectDirectoryButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Directory</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>749</x>
      <y>320</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="directoryNameWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>1021</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="pathListView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>711</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startAnalysisButton">
    <property name="geometry">
     <rect>
      <x>969</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze folder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
